helm/coredns-app/templates/rbac.yaml

```diff
@@ -13,6 +13,7 @@ rules:
   - services
   - pods
   - namespaces
+  - endpointslices
```

Comment:
You'll probably have to put this in the `discovery.k8s.io` api group

Reply:
yep :) thank you for pointing that out.